assets/training/finetune_acft_image/components/pipeline_components/object_detection/spec.yaml

```diff
@@ -379,6 +379,20 @@ jobs:
       warmup_cosine_lr_warmup_epochs: ${{parent.inputs.warmup_cosine_lr_warmup_epochs}}
       weight_decay: ${{parent.inputs.weight_decay}}
 
+  finetune_common_validation:
+    type: command
+    component: azureml:finetune_common_validation:0.0.2
+    compute: ${{parent.inputs.compute_model_import}}
+    inputs:
+      mlflow_model_path: ${{parent.inputs.mlflow_model}}
+      train_mltable_path: ${{parent.inputs.training_data}}
+      validation_mltable_path: ${{parent.inputs.validation_data}}
+      compute_model_import: ${{parent.inputs.compute_model_import}}
+      compute_finetune: ${{parent.inputs.compute_finetune}}
+      task_name: ${{parent.inputs.task_name}}
+      user_column_names: image_url, label
+      task_specific_extra_params: '"model_family=${{parent.inputs.model_family}};model_name=${{parent.inputs.model_name}};metric_for_best_model=${{parent.inputs.metric_for_best_model}};number_of_epochs=${{parent.inputs.number_of_epochs}}"'
+
```

Comment:
Add validation_output: ${{parent.jobs.finetune_common_validation.outputs.validation_info}} to model_import as input.

Reply:
Also, could you please change common_validation_comp syntax from registry to workspace asset for hf_classification and mmd_od_is.